web/impact/impact/v1/views/mentor_program_office_hour_list_view.py

```diff
@@ -45,6 +47,14 @@ def _filter_by_user_open_hours(self, qs):
         qs = qs.filter(program_id__in=program_ids, finalist__isnull=True)
         return qs
 
+    def _filter_by_user_all_hours(self, qs):
```

Comment:
I'm not seeing how this is related to this ticket

Reply:
Some other ticket's work. Removing thanks